src/test/ui/borrowck/issue-64453.stderr

```diff
@@ -5,28 +5,37 @@ LL |     let settings_data = from_string(settings_dir);
    |                                     ^^^^^^^^^^^^ move occurs because `settings_dir` has type `std::string::String`, which does not implement the `Copy` trait
 
 error[E0019]: static contains unimplemented expression type
-  --> $DIR/issue-64453.rs:4:31
+  --> $SRC_DIR/liballoc/macros.rs:LL:COL
    |
-LL | static settings_dir: String = format!("");
-   |                               ^^^^^^^^^^^
+LL |     ($($arg:tt)*) => ($crate::fmt::format($crate::__export::format_args!($($arg)*)))
```

Comment:
This... seems not great. I don't think it helps people to see this, and in fact is probably more confusing for the average person?

Reply:
Not sure -- ostensibly it could be more helpful because it highlights the reason behind the error...

Reply:
I don't get any more information to be honest from this underlying function/macro and it's sort of quite "raw" and such